x-pack/filebeat/input/httpjson/input_test.go

```diff
@@ -1235,8 +1239,8 @@ func TestInput(t *testing.T) {
 					}
 				}
 			}
-			if tc.baseConfig["verifyfilepath"] != nil {
-				if _, err := os.Stat("../../logs/httpjson/http-request-trace-httpjson-foo-eb837d4c-5ced-45ed-b05c-de658135e248_https_somesource_someapi.ndjson"); err == nil {
+			if len(tc.expectedfile) > 0 {
+				if _, err := os.Stat(tc.expectedfile); err == nil {
```

Comment:
In order to be able to use `t.TempDir()` for test isolation let's treat all `expectedfile` values as relative to the that location. So

```suggestion
				if _, err := os.Stat(filepath.Join(t.TempDir(), tc.expectedfile)); err == nil {
```